### DISCUSSION TITLE
ConvexSpheropolyhedronUnion visualization in Ovito

### DISCUSSION THREAD

**Discussion by e23j** [glotzerlab/hoomd-blue#1749] [Support] [answered]
Is there a way to log the shape of the particles in the gsd file using the ConvexSpheropolyhedronUnion integrator to be able to visualize the shapes of the particles in the union using Ovito?

**Comment by tcmoore3** [comment, marked as answer]
Not in HOOMD, no. But you can either 1) write a script that reads your gsd file and adds extra particles so that each shape of the union is a separate particle and include the shape information in the file, or 2) create a .obj file with your shape union and load that into ovito.
